.github/workflows/test.yml

```diff
@@ -8,19 +8,6 @@ on:
   workflow_dispatch:
 
```

Comment:
we need to remove some parts in goreleaser file as well, like `builds:` section (need to check other as well though)